apps/sms/views/conversation/js/conversation.js

```diff
@@ -516,6 +527,8 @@ var ConversationView = {
   beforeEnter: function conv_beforeEnter(args) {
     this.clearConvertNoticeBanners();
     this.setHeaderAction(ActivityClient.hasPendingRequest() ? 'close' : 'back');
+    this.header.removeAttribute('no-font-fit');
```

Comment:
> only first time and it's then noop; would you like a comment for this?

Yeah, at least for "me in the future" it will be easier to understand with comment :) Thanks!

> I think it's faster to just call removeAttribute than having a condition with hasAttribute..

Agree